.circleci/config.yml

```diff
@@ -105,8 +105,9 @@ executors:
       # Set cluster workers to 5 (5 cluster nodes)
       # NUM_NODES represent number of parallel test executions
       # NUM_WORKERS represent number of nodes in a k8 cluster
-      NUM_NODES: 8
-      NUM_WORKERS: 10
+      NUM_NODES: 5 
```

Comment:
Curious on how this made a difference? Do we not have access to enough nodes/workers?

Reply:
The value was lower earlier, and we were experimenting to see if bumping up the nodes would reduce the run time of tests. Apparently this is now causing resource cleanup issues